services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/mapper/annotations/DynamoDbBean.java

```diff
@@ -39,5 +40,13 @@
 @Retention(RetentionPolicy.RUNTIME)
 @SdkPublicApi
 public @interface DynamoDbBean {
-    Class<? extends AttributeConverterProvider>[] converterProviders() default {};
+    Class<? extends AttributeConverterProvider>[] converterProviders()
+            default { NoAttributeConverterProviderOverride.class };
+
+    final class NoAttributeConverterProviderOverride implements AttributeConverterProvider {
```

Comment:
By inheritance, this class is effectively public (which is why you're able to use it elsewhere in the code). That means we should either javadoc it or move it out of here and into internal namespace. I think I favor the latter but open to discussion.

Reply:
Offline agreed to just use DefaultAttriburteConverterProvider instead (which will be a public class in this PR)